spec/models/transaction_spec.rb

```diff
@@ -14,10 +14,24 @@
     it { should validate_presence_of(:providing_organisation_type) }
     it { should validate_presence_of(:receiving_organisation_name) }
     it { should validate_presence_of(:receiving_organisation_type) }
-  end
 
-  describe "associations" do
-    it { should belong_to(:report) }
+    context "when the activity belongs to a delivery partner organisation" do
+      before { activity.update(organisation: build_stubbed(:delivery_partner_organisation)) }
```

Comment:
(TIL) I hadn't seen this `build_stubbed` used before. I've read about it now and it seems a good idea to use it here.

Reply:
Yeah, I wish I was better at A: remembering to use it and B: had a little more depth to my understanding of it and when to use it!